src/main/java/com/github/monster860/fastdmm/FastDMM.java

```diff
@@ -805,4 +870,59 @@ private void loop() {
 			Display.sync(60);
 		}
 	}
+
+	private String getAppData() {
```

Comment:
Well, it's done now. I left the function in because of folder creation and the complications about paths on different OSs, not to clutter the other functions.